packages-tests/Caching/Detector/ChangedFilesDetectorTest.php

```diff
@@ -33,9 +33,6 @@ public function testHasFileChanged(): void
         $this->changedFilesDetector->addFileWithDependencies($smartFileInfo, []);
 
         $this->assertFalse($this->changedFilesDetector->hasFileChanged($smartFileInfo));
-        $this->changedFilesDetector->invalidateFile($smartFileInfo);
-
-        $this->assertTrue($this->changedFilesDetector->hasFileChanged($smartFileInfo));
```

Comment:
This seems to kill the original purpose of the test. 
It should be kept, even if with different code.